dasharo-compatibility/esp-scanning.robot

```diff
@@ -27,6 +27,7 @@ Suite Setup         Run Keywords
 ...                     AND
 ...                     Prepare EFI Partition With System Files
 Suite Teardown      Run Keywords
+...                     Power On    AND
```

Comment:
`Power On` is already in the `Clear Out EFI Partition` - seems redundant.

Reply:
https://github.com/Dasharo/open-source-firmware-validation/blob/develop/lib/esp-scanning-lib.robot#L65